src/libstd/io/mod.rs

```diff
@@ -449,6 +443,32 @@ fn read_to_end<R: Read + ?Sized>(r: &mut R, buf: &mut Vec<u8>) -> Result<usize>
 /// # Ok(())
 /// # }
 /// ```
+///
+/// Read from `&str` because [`&[u8]`] implements [`Read`]:
```

Comment:
Pulldown doesn't seem to able to parse ``[`&[u8]`]`` (@steveklabnik please confirm)

* https://doc.rust-lang.org/nightly/std/io/trait.Read.html Broken

```
 Documenting std v0.0.0 (file:///home/nodakai/src/rust-HEAD/src/libstd)
WARNING: documentation for this crate may be rendered differently using the new Pulldown renderer.
    See https://github.com/rust-lang/rust/issues/44229 for details.
WARNING: rendering difference in `The `Read` trait allows for reading bytes from a source.`
   --> src/libstd/io/mod.rs:470:0
    /html[0]/body[1]/p[8] Text differs:
        expected: `because`
        found:    `because [`
WARNING: rendering difference in `The `Read` trait allows for reading bytes from a source.`
   --> src/libstd/io/mod.rs:470:0
    /html[0]/body[1]/p[8] Tags differ: expected: `a`, found: `code`
WARNING: rendering difference in `The `Read` trait allows for reading bytes from a source.`
   --> src/libstd/io/mod.rs:470:0
    /html[0]/body[1]/p[8] Text differs:
        expected: `implements`
        found:    `] implements`
WARNING: rendering difference in `The `Read` trait allows for reading bytes from a source.`
   --> src/libstd/io/mod.rs:470:0
    /html[0]/body[1] Unexpected element `p`: found: `<p>[<code>&amp;[u8]</code>]: primitive.slice.html</p>`
```